app/views/mail/jobFailedUploadConvert.scala.html

```diff
@@ -0,0 +1,26 @@
+@(name: String, datasetName: String)
+
+@emailBaseTemplate() {
+<p>Hi @{name},</p>
+
+<p>
+  oops, something went unexpected wrong when uploading and converting your dataset <i>@{datasetName}</i> for WEBKNOSSOS. 
```

Comment:
Maybe something like "unfortunately, your uploaded dataset could not be converted automatically for WEBKNOSSOS" would sound a bit more professional?

Reply:
Done